src/main/java/decodes/cwms/CwmsSiteDAO.java

```diff
@@ -488,29 +494,38 @@ public void fillCache()
 				{
 					warning("SiteName for id=" + key + " (" + nameType + ":"
 						+ nameValue + ") but no matching site.");
-					continue;
+					return;
 				}
 				SiteName sn = new SiteName(site, nameType, nameValue);
 				sn.setUsgsDbno(rs.getString(4));
 				sn.setAgencyCode(rs.getString(5));
 				site.addName(sn);
-				nNames++;
-			}			
+				nNames[0]++;
+			}, parameters.toArray(new Object[0]));
+
+			setFetchSize(origFetchSize);
```

Comment:
I can't see where FetchSize is being used.

Reply:
It's set higher in the function, this is returning it back... actually think that might need to be outside the catch block.